api/src/main/resources/workbench.yaml

```diff
@@ -484,7 +484,7 @@ paths:
           type: string
           required: false
 
-  /v1/account/bulkSyncTrainingStatus:
+  /v1/cron/bulkSyncTrainingStatus:
```

Comment:
Could you also move this down to 812? There's a relevant comment header there https://github.com/all-of-us/workbench/blob/7cc0a4b0275ae21b0da63b38f67d761b5202cdf0/api/src/main/resources/workbench.yaml#L812